charts/splunk-connect-for-snmp/templates/worker/trap/deployment.yaml

```diff
@@ -59,6 +59,23 @@ spec:
               readOnly: false
           resources:
             {{- toYaml .Values.worker.trap.resources | nindent 12 }}
+      {{- if .Values.worker.trap.livenessProbe }}
+      livenessProbe:
```

Comment:
you don't need to use toYaml everywhere, just cast .Values.worker.trap.livenessProbe toYaml